.cirrus.yml

```diff
@@ -95,6 +95,13 @@ task:
         - fi
         - export CIRRUS_CHANGE_MESSAGE=`cat /tmp/cirrus_change_message.txt`
         - export CIRRUS_COMMIT_MESSAGE=`cat /tmp/cirrus_commit_message.txt`
+    - name: build-linux+drive-examples
+      install_script:
+        - flutter config --enable-linux-desktop
+      build_script:
```

Comment:
Should we run the tests on stable, too? 
Or it is in the future plan, can we add a TODO here?

Reply:
Like web, stable isn't supported for Linux. I'll add a TODO.